runtime/doc/map.txt

```diff
@@ -1358,6 +1358,20 @@ Possible attributes are:
 Note that -range=N and -count=N are mutually exclusive - only one should be
 specified.
 
+                                                  *E889* *:command-addr*
+It is possible that the special characters in the range like ., $ or % which
```

Comment:
Perhaps the mentioned special characters should be surrounded by double quotes or backtics. Stuff like `range like ., $` are odd to read IMO.